### PR TITLE
Study tool

### PR DESCRIPTION
**Pull request by mwitte98**
Related Issue #30

Changes:
- Study tool

### CODE REVIEW

File: client/src/components/Lesson/WordCreator/WordRootSelector.tsx [hunk 1]
```diff
@@ -54,35 +52,30 @@ const WordRootSelector = (props: WordRootSelectorProps) => {
...
-                onChange={onRootChange}
-                disabled={props.disabled} />
+                onChange={onRootChange} />
             <TextField
                 name={root.meaning}
                 value={root.meaning}
                 hintText='Root meaning'
                 style={{ width: '75%', marginLeft: '5%' }}
-                onChange={onMeaningChange}
-                disabled={props.disabled} />
```
Comment (7imbrook) [inline client/src/components/Lesson/WordCreator/WordRootSelector.tsx:79]: Why are we removing disabled? We'd have 3 states for this, editing, editing+locked and, study view?

Comment (mwitte98) [reply client/src/components/Lesson/WordCreator/WordRootSelector.tsx:79]: Editing+locked is now the study view so we'll have 2 states: editing and study view

Comment (7imbrook) [reply client/src/components/Lesson/WordCreator/WordRootSelector.tsx:79]: is that what we want? I guess thats ok.

Comment (mwitte98) [reply client/src/components/Lesson/WordCreator/WordRootSelector.tsx:79]: I think it's better without all of the disabled inputs

cc @djrenren

**Review by 7imbrook — APPROVED** [review]

**Pull request merged**